packages/backend/src/api/endpoints/AccountlessApplicationsAPI.ts

```diff
@@ -5,17 +5,26 @@ import { AbstractAPI } from './AbstractApi';
 const basePath = '/accountless_applications';
 
 export class AccountlessApplicationAPI extends AbstractAPI {
-  public async createAccountlessApplication() {
+  public async createAccountlessApplication(headers: Headers | undefined) {
```

Comment:
Let's change to optional object instead

```suggestion
  public async createAccountlessApplication(params?: { requestHeaders?: Headers }) {
```